digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/pointasset/OracleObstacleDao.scala

```diff
@@ -83,6 +76,8 @@ object OracleObstacleDao {
   }
 
   def assetRowToProperty(assetRows: Iterable[ObstacleRow]): Seq[Property] = {
+    val specialCases = Seq("opposite_side_sign", "suggest_box")
```

Comment:
We have this property "opposite_side_sign" in the obstacle asset type? We shouldn't have this one, right?

Reply:
Done